polyply/tests/test_topology.py

```diff
@@ -627,7 +670,8 @@ def test_replace_types(lines, outcome):
         polyply.src.top_parser.read_topology(new_lines, top)
         top.gen_bonded_interactions()
         for inter_type in outcome:
-            assert top.molecules[0].molecule.interactions[inter_type] == outcome[inter_type]
+            for inter in top.molecules[0].molecule.interactions[inter_type]:
+                assert top.molecules[0].molecule.interactions[inter_type] == outcome[inter_type]
```

Comment:
This looks wrong, you execute the same assert multiple times

Reply:
Yep that seems strange indeed